src/blocks/ExtendedFeatures/ExtendedFeatures.tsx

```diff
@@ -38,6 +38,7 @@ export const ExtendedFeaturesBlock = ({
                         ({
                             title: itemTitle,
                             text,
+                            list,
```

Comment:
how does it work with no changes in types?

Reply:
’ExtendedFeaturesItem’ already use ’Omit<ContentBlockProps, >’